### DISCUSSION TITLE
Classes for `flex-direction` are missing in bundled CSS

### DISCUSSION THREAD

**Discussion by kwiat1990** [tailwindlabs/tailwindcss#11149] [Help] [answered]
**What version of Tailwind CSS are you using?**

For example: v3.2.2

**What build tool (or framework if it abstracts the build tool) are you using?**

For example: Remix 1.16.0 with an express server

**What version of Node.js are you using?**

For example: v16.18.1

**What browser are you using?**

For example: Chrome

**What operating system are you using?**

For example: macOS

**Reproduction URL**
Considering following JSX/HTML
```tsx
<div className="flex flex-row-reverse">
    <div>1</div>
    <div>2</div>
</div>
```
`.flex-row-reverse` class won't be included in the CSS files loaded in the browser.

**Describe your issue**

None of the [classes](https://tailwindcss.com/docs/flex-direction) related to`flex-direction` is included in the bundled Tailwind CSS file.

**Comment by wongjn** [comment, marked as answer]
Consider checking:
- Tailwind has compiled since you added the class.
- The `content` file globs sufficiently cover the file that the code appears in.
- The `flexDirection` core Tailwind plugin has not been disabled

Otherwise if you have checked all the above, please consider providing a remotely-hosted git repository that reproduces the unexpected behavior.

**Reply by adamwathan** [reply]
Just to add to what @wongjn said, here's a Play that shows the classes are generated, so it's something specific to your project:

https://play.tailwindcss.com/Lz7wEMB8x0

![image](https://user-images.githubusercontent.com/4323180/235925370-625b5dbf-0cf7-4cbb-87fc-e81b67352597.png)

**Reply by kwiat1990** [reply]
@wongjn thanks for pointing this out. I have updated Remix to the latest version, which also changes a bit how Tailwind can be included in the project and I've moved input`tailwind.css` file around but it turned out to the wrong place (root of the app but not in the `app` (Remix's `src`) directory). It worked because of generated `tailwind.css` file was still at the place. But new classes weren't added. Now everything works as it should. Sorry for the "noise".
